### DISCUSSION TITLE
Detecting Draggable collision?

### DISCUSSION THREAD

**Discussion by Logaqm** [clauderic/dnd-kit#275] [Q&A]
Is there any pre-existing solution for detecting collision between two draggables? This use case does not seem to be covered, but I figured I would ask before taking the time to build my own solution, since the documentation seems to only be referring to collision between draggables and droppables.

**Comment by clauderic** [comment]
Not really sure what use case you have in mind, but have you considered making your draggable nodes both draggable and droppable, similar to how the sortable preset works?

**Reply by Logaqm** [reply]
My use case is pretty straight forward- when one draggable is dragged over another i want to raise the z index above that one so it appears on top. That should do the trick.  [video](https://gyazo.com/b8240f5d7d9e167ef134b0eab4fb4922).

**Comment by sidonaldson** [comment]
I have a similar query if anyone could give some advice?
I'm using draggable to move hotspots around on an image but I'd love the hotspots to not collide or overlap.
Is this possible without using the dragEnd handler to manually calculate overlap?

@Logaqm did you do something similar?

**Comment by maddy302** [comment]
I have a similar use case as above. Create hotspots on an image and not let them overlap.
Since dndkit is providing droppable elements, can we have a prop that make these draggable elements not overlap/collide? 

How can I request for an enhancement? Is there an issue already open ?

**Reply by maddy302** [reply]
@clauderic  Sry for the mention, there were no replies on a similar usecase above and hence requesting your attention.
